### ISSUE TITLE
jaxrs-raml-maven-plugin:generate-raml generates invalid RAML for API with wildcard

### ISSUE THREAD

**Issue by dkirrane** [mulesoft/raml-for-jax-rs#91]
I have a JAX-RS REST API like:

```
    @GET
    @Consumes(MediaType.WILDCARD)
    @Produces(MediaType.TEXT_HTML)
    @Path("/call")
    public Response call() {
    }
```

jaxrs-raml-maven-plugin:generate-raml generates an invalid RAML file for this 

```
/service/call: 
  get: 
    body: 
      wildcard: 
    responses: 
      200: 
        body: 
          text/html: 
```

RAML Console complains with `property: 'wildcard' is invalid in a body`

**Comment by KonstantinSviridov** [comment]
Hi, @dkirrane 

I suppose, the `MediaType.WILDCARD` constant has `wildcard` value. If it really does, then the generator works correctly and you should ask RAML Console team (or RAML Parser team) about supported media types. 

Regerds, Konstantin.

**Comment by dkirrane** [comment]
Hi, @KonstantinSviridov

Class `javax.ws.rs.core.MediaType` has:
```
public static final String WILDCARD = "*/*";
```
I replaced  `body:  wildcard:` with  `body: "*/*"` in my raml file and it gets displayed correctly in RAML console.

**Comment by KonstantinSviridov** [comment]
Ok. That looks like a bug. I'll tak a look at it.

**Comment by KonstantinSviridov** [comment]
So, the problem is that we do not support expressions as annotation values now. Even such simple as constants. We just examine the string value (in your case it is "MediaType.TEXT_HTML" and "MediaType.WILDCARD") and try to detect the mediatype. The first constant name has two "good" substrings: `text` and `html`, so the media type is recognised.

**Comment by KonstantinSviridov** [comment]
Hi, @dkirrane 

The fix is in 1.3.4-SNAPSHOT

Regards, Konstantin
